JSQMessagesViewController/Views/JSQMessagesComposerTextView.h

```diff
@@ -18,6 +18,21 @@
 
 #import <UIKit/UIKit.h>
 
+@class JSQMessagesComposerTextView;
+
+/**
+ * A delegate used to forward custom notifications from `JSQMessagesComposerTextView`.
+ */
+@protocol JSQMessagesComposerTextViewPasteDelegate <NSObject>
+
+/**
+ * Asks the delegate whether or not the `textView` should use the original implementation of -[UITextView paste].
+ * If you want any custom pasting behavior, implement this delegate method and return `NO`
+ * when you want to handle pasting yourself.
```

Comment:
1. Wrap "-[UITextView paste]" in ticks \` \`

2. Second sentence should become:
> @discussion Use this delegate method to implement custom pasting behavior. You should return `NO` when you want to handle pasting. Return `YES` to defer functionality to the textView.